benchmarks/ccsynchbench.c

```diff
@@ -49,17 +52,16 @@ int main(int argc, char *argv[]) {
     object->state_f = 1.0;
     CCSynchStructInit(object_combiner, bench_args.nthreads);
 
-    BarrierInit(&bar, bench_args.nthreads);
+    BarrierSet(&bar, bench_args.nthreads);
     StartThreadsN(bench_args.nthreads, Execute, bench_args.fibers_per_thread);
     JoinThreadsN(bench_args.nthreads - 1);
-    d2 = getTimeMillis();
 
     printf("time: %d (ms)\tthroughput: %.2f (millions ops/sec)\t", (int)(d2 - d1), bench_args.runs * bench_args.nthreads / (1000.0 * (d2 - d1)));
     printStats(bench_args.nthreads, bench_args.total_runs);
 
 #ifdef DEBUG
-    fprintf(stderr, "DEBUG: object state: %f\n", object->state_f);
-    fprintf(stderr, "DEBUG: object counter: %d\n", object_combiner->counter);
+    fprintf(stderr, "DEBUG: Object float state: %f\n", object->state_f);
+    fprintf(stderr, "DEBUG: Object state: %ld\n", object_combiner->counter);
```

Comment:
So the state is the counter?

Reply:
In this case, we can use the counter in order to track the number of operations applied in the object.
Since, the object is a Fetch&Multiply object, its state (a float value) may mean nothing.